### PR TITLE
{math,phys}[foss/2019b] georges v2019.2 + LMfit v1.0.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by ekaterinaasf**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/LMfit/LMfit-1.0.0-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('asteval', '0.9.17', {
```
Comment (boegel) [inline easybuild/easyconfigs/l/LMfit/LMfit-1.0.0-foss-2019b-Python-3.7.4.eb:22]: @ekaterinaasf The existing `LMfit` easyconfig using `intel/2019b` as toolchain uses `asteval` 0.9.18, so I see no reason not to do that here too?

**Review by boegel — CHANGES_REQUESTED** [review]
@ekaterinaasf Thank you very much for your contribution, and welcome to EasyBuild!

It looks like `LMfit-1.0.1-foss-2020a-Python-3.8.2.eb` is being removed by accident here?

If you need help with restoring it, do let us know!

**Comment by verdurin** [comment]
@boegel Are you happy with this now?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2725.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/de2d083c4a939387f4f3a67c308056f9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3523.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/6f51acfc7b9d2595e02f22376c8df554 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ekaterinaasf!

**Pull request merged**